examples/oauth-proxy/readme.md

```diff
@@ -0,0 +1,70 @@
+# OAuth Proxy APIm policies
```

Comment:
Can you add an entry in https://github.com/Azure/api-management-policy-snippets/blob/master/examples/README.md please? We list every policy scenario there so we can remove this README here and align with existing approach

Reply:
Done